stories/examples/Forms.stories.mdx

```diff
@@ -0,0 +1,105 @@
+<Meta title="Examples|Forms" />
+
+# Forms
+
+```jsx
+() => {
+  const [username, setUsername] = React.useState('');
+  const [usernameError, setUsernameError] = React.useState('');
+
+  const [password, setPassword] = React.useState('');
+  const [passwordError, setPasswordError] = React.useState('');
+
+  return (
+    <Flex bg="gray.200" py="40px" px="12px" justifyContent="center" alignItems="center">
```

Comment:
Matt, the padding of 12px being added to the left and right on <Flex> doesn't appear to make a difference when I remove it. Maybe it's not necessary?

Reply:
Ah, good catch! Yeah I don't think that's needed anymore. Thanks!